README.md

```diff
@@ -36,121 +40,45 @@ output "postgresql_service_uri" {
 }
 ```
 
-- Run in your terminal:
-```bash
-$ terraform init
-$ terraform plan
-$ terraform apply
-$ psql "$(terraform output -raw postgresql_service_uri)"
-```
-
-Voilà, a PostgreSQL database.
-
-## A word of caution
-Recreating stateful services with Terraform will possibly **delete** the service and all its data before creating it again. Whenever the Terraform plan indicates that a service will be **deleted** or **replaced**, a catastrophic action is possibly about to happen.
-
-Some properties, like **project** and the **resource name**, cannot be changed and it will trigger a resource replacement.
-
-To avoid any issues, **please set the `termination_protection` property to `true` on all production services**, it will prevent Terraform to remove the service until the flag is set back to `false` again. While it prevents a service to be deleted, any logical databases, topics or other configurations may be removed **even when this section is enabled**. Be very careful! 
+- Run these commands in your terminal:
 
-## Developing
-
-### Requirements
-- [Terraform](https://www.terraform.io/downloads.html) v0.12.X or greater
-- [Go](https://golang.org/doc/install) 1.16.X or greater
-
-### Testing
-Run the tests with the command below:
-```bash
-$ make test
-```
-
-Run the acceptance tests with the commands below:
 ```bash
-$ export AIVEN_TOKEN="your-token"
-$ export AIVEN_PROJECT_NAME="your-project-name"
-
-$ make testacc
-
-# or run a specific acceptance test
-$ make testacc TESTARGS='-run=TestAccAiven_kafka'
-```
-
-> Acceptance tests create real resources, and often cost money to run.
-
-For information about writing acceptance tests, see the main [Terraform contributing guide](https://www.terraform.io/docs/extend/testing/acceptance-tests/index.html).
-
-### Alternate command to build/test inside a container
-
-In case you'd like to avoid Go-dependency on your local machine or CI, you can run the following build command:
-
-```
-podman run -v .:/terraform-provider-aiven:z --workdir /terraform-provider-aiven golang:latest make <COMMAND>
+terraform init && \
+    terraform plan && \
```

Comment:
Also, please remove `&& \` because `terraform apply` requires user action, and the whole idea of having `terraform plan` does not make sense in this sequence.